packages/client/ui/react-ui/src/hooks/useCrossmint.tsx

```diff
@@ -16,14 +17,17 @@ export function CrossmintProvider({
     const [version, setVersion] = useState(0);
 
     const crossmintRef = useRef<Crossmint>(
-        new Proxy<Crossmint>(createCrossmint(createCrossmintParams), {
-            set(target, prop, value) {
-                if (prop === "jwt" && target.jwt !== value) {
-                    setVersion((v) => v + 1);
-                }
-                return Reflect.set(target, prop, value);
-            },
-        })
+        new Proxy<Crossmint>(
+            createCrossmint({ ...createCrossmintParams, jwt: createCrossmintParams.jwt ?? getCachedJwt() }),
```

Comment:
So that the JWT is included in the first render

Reply:
If a client using external auth passes in a jwt, that will be used. Otherwise, we we check if there's an existing session on the browser for crossmint auth, otherwise jwt is undefined initially.